src/Features/Core/Portable/QuickInfo/CommonQuickInfoContext.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+
+namespace Microsoft.CodeAnalysis.QuickInfo
+{
+    internal readonly struct CommonQuickInfoContext
+    {
+        public readonly Workspace Workspace;
```

Comment:
note: this is just used to retrieve language services.  it coudl be weakened to that type if desired.